.github/workflows/main.yml

```diff
@@ -423,6 +423,9 @@ jobs:
     name: Integration test (jsii-pacmak)
     runs-on: ubuntu-latest
     needs: create-release-package
+    strategy:
+      matrix:
+        rosetta: ['1.x', '5.2.x', '5.3.x', '5.4.x']
```

Comment:
Unfortunately the workflow file is not codegen'd, so we will have to update this manually.